src/connector/src/sink/iceberg.rs

```diff
@@ -201,8 +204,7 @@ impl IcebergSink {
 
 #[async_trait::async_trait]
 impl Sink for IcebergSink {
-    /// TODO: Can't support commit coordinator now.
-    type Coordinator = DummySinkCommitCoordinator;
+    type Coordinator = IcebergSinkCommitter;
     type Writer = IcebergWriter;
```

Comment:
Should we wrap it with CoordinatedSinkWriter if we want to use a seperate Coordinator?

Reply:
I think it's already wrapped in https://github.com/risingwavelabs/risingwave/blob/2342e8b8fc036c004e6935628e790ef007a6d6c7/src/connector/src/sink/coordinate.rs#L24

Reply:
It's not wrapped yet. It should be `type Writer = CoordinatedSinkWriter<IcebergWriter>`. The `CoordinatedSinkWriter` is responsible for communication with the coordinator.

Reply:
Fixed. But this design doesn't follow DRY principal, but we can improve it later.